src/style-sheets/layout/_toolbar.scss

```diff
@@ -21,6 +21,10 @@
 
       list-style-type: none;
 
+      & li:hover {
```

Comment:
sorry it wasn't specified but the links for this PR do not include the links in the toolbar. Please remove the css from this file